scripts/artifactory-cleanup.js

```diff
@@ -0,0 +1,69 @@
+const exec = require('child_process').exec;
+const semver = require('semver');
+const packageJson = require('../package.json');
+
+const BASE_ARTIFATORY_URL = 'https://artifactory.wikia-inc.com/artifactory';
+const BASE_PACKAGE_PATH = '/npm-local/@fandom/jwplayer-fandom/-/@fandom';
+const BASE_READ_PATH = `/api/storage${BASE_PACKAGE_PATH}`;
+const BASE_DELETE_PATH = BASE_PACKAGE_PATH;
+
+const currentMajorVersion = semver.major(packageJson.version);
+const previousMinorVersion = semver.minor(packageJson.version) - 1;
+const versionThreshold = `${currentMajorVersion}.${previousMinorVersion}.0`;
```

Comment:
let's remove all test versions lower than current version - 1 minor version:
e.g. if the current version in `2.6.4` we will remove all test packages that are marked as `2.4.X` and lower (we'll keep `2.6.X` and `2.5.X`).

Reply:
Should we deprecate versions before removing them? npm has a command for this (`npm deprecate <package-name>@<version> "<message>"`), not sure about Yarn.

Reply:
I think it's not needed as we are going to remove them sooner rather than later. No one will notice that these packages are deprecated as no one is using them anymore.

Reply:
Okay, I wasn't sure about their current usage. According to @nandy-andy, we don't have a full list of dependents at the moment.

Reply:
That's true. However, it's very likely that the only consumers here are UCP and NAR